lib/teach-api.js

```diff
@@ -230,6 +230,12 @@ _.extend(TeachAPI.prototype, {
         ga.event({ category: 'Clubs', action: 'Deleted a Club' });
         callback(null);
       }.bind(this));
+  },
+  getClubsGuides: function(callback) {
+    callback = callback || function () {};
```

Comment:
we introduced ES6 well after the entire codebase was set up, at this point it's easier to update the bits we touch to ES6 as we touch them rather than converting entire files at a time.